From 9ec65c5a19dfd130003b79a8f63d60ea8e39e533 Mon Sep 17 00:00:00 2001
From: Todd Whiteman <todd.whiteman@joyent.com>
Date: Fri, 26 Apr 2019 10:52:12 -0700
Subject: [PATCH] TRITON-1638 migration test for uuid override is failing with
 "vm uuid should be different" Reviewed by: Trent Mick <trentm@gmail.com>
 Approved by: Trent Mick <trentm@gmail.com>

---
 test/lib/migration.js | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/test/lib/migration.js b/test/lib/migration.js
index 2580a6c..0fc334c 100644
--- a/test/lib/migration.js
+++ b/test/lib/migration.js
@@ -1112,13 +1112,9 @@ function TestMigrationCfg(test, cfg) {
                 if (migrationUuidOverride) {
                     t.equal(vm.state, 'destroyed',
                         'original vm should have state destroyed');
-                    t.notEqual(sourceVm.uuid, vm.uuid,
-                        'vm uuid should be different');
                     t.equal(sourceVm.server_uuid, vm.server_uuid,
                         'vm server_uuid should be the same');
                 } else {
-                    t.equal(sourceVm.uuid, vm.uuid,
-                        'vm uuid should be the same');
                     t.notEqual(sourceVm.server_uuid, vm.server_uuid,
                         'vm server_uuid should be different');
                 }
-- 
2.21.0

